src/app/settings/action-triggers/components/action-triggers-list/action-triggers-list.component.ts

```diff
@@ -0,0 +1,241 @@
+  ]
+  public selectMode: boolean = false
+  private readonly selectedActionTriggerIds: Set<string> = new Set()
+
+  constructor(
+    private readonly dialogService: DialogService,
+    private readonly fileDownloadService: HttpFileDownloadService,
+    private readonly actionTriggerService: ActionTriggerService
+  ) {}
+
+  public ngOnChanges(changes: SimpleChanges): void {
+    const actionTriggerChange: SimpleChange | undefined = changes['actionTriggers']
+    if (actionTriggerChange) {
+      this.newSortSelect(this.sort)
+    }
+  }
+
+  get filteredActionsTriggers(): ActionTriggerWithActionInfo<KeyboardTriggerData>[] {
+    return this.actionTriggers.filter(
+      trigger =>
+        trigger.data.label.toLocaleLowerCase().includes(this.search.toLocaleLowerCase()) ||
+        trigger.data.keys.toString().toLocaleLowerCase().includes(this.search.toLocaleLowerCase()) ||
+        trigger.id === this.selectedActionTrigger?.id
+    )
+  }
+
+  public selectActionTrigger(actionTrigger: ActionTriggerWithActionInfo<KeyboardTriggerData> | undefined): void {
+    this.onActionTriggerOpen.emit(this.selectedActionTrigger?.id === actionTrigger?.id ? undefined : actionTrigger)
+  }
+
+  public setActionTriggerSelection(isSelected: boolean, id: string): void {
+    if (isSelected) {
+      this.selectedActionTriggerIds.add(id)
+    } else {
+      this.selectedActionTriggerIds.delete(id)
+    }
+    this.updateSelectedActionTriggersOptions()
+  }
+
+  public actionTriggerDelete(actionTriggerId: string): void {
+    this.dialogService.createConfirmDialog($localize`global.delete.label`, $localize`action-triggers.delete.confirmation`, 'Delete', () => this.deleteActionTriggerById(actionTriggerId))
+  }
+
+  private deleteActionTriggerById(actionTriggerId: string): void {
+    this.closeSelectedTabIfAreDeleted(actionTriggerId)
+    this.actionTriggerService.deleteActionTrigger(actionTriggerId).subscribe()
+  }
+
+  private closeSelectedTabIfAreDeleted(deletedActionId: string): void {
+    if (this.selectedActionTrigger?.id === deletedActionId) {
+      this.onActionTriggerOpen.emit(undefined)
+    }
+  }
+
+  private deleteSelectedActionTriggers(): void {
+    this.selectedActionTriggerIds.forEach(actionTriggerId => this.deleteActionTriggerById(actionTriggerId))
+  }
+
+  public actionTriggerCopy(actionTrigger: ActionTriggerWithActionInfo<KeyboardTriggerData>): void {
+    const copyPayload: ActionTrigger<KeyboardTriggerData> = {
+      actionId: actionTrigger.actionId,
+      id: '',
+      data: {
+        keys: actionTrigger.data.keys,
+        label: actionTrigger.data.label,
+        actionArguments: actionTrigger.data.actionArguments,
+        triggerOn: actionTrigger.data.triggerOn,
+      },
+    }
+    if (actionTrigger.data.mappedToKeys && actionTrigger.data.mappedToKeys.length > 0) {
+      copyPayload.data.mappedToKeys = actionTrigger.data.mappedToKeys
+    }
+    this.actionTriggerService.createActionTrigger(copyPayload).subscribe()
+  }
```

Comment:
I use `ActionTriggerWithActionInfo`, because actually this is comes by the function when it is called form the template, because the actionTriggers sheet is of this type
`@Input() public actionTriggers: ActionTriggerWithActionInfo<KeyboardTriggerData>[]`